src/test/java/gomedic/model/person/patient/PatientTest.java

```diff
@@ -0,0 +1,135 @@
+package gomedic.model.person.patient;
+
+import static gomedic.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import gomedic.model.commonfield.Name;
+import gomedic.model.commonfield.Phone;
+
+public class PatientTest {
```

Comment:
I think you might want to make a PatientBuilder class, then use it to create a list of typical patients or sth (can refer to PersonBuilder / Doctor Builder)